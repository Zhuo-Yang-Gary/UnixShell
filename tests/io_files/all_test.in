echo my own "\"test\"" case which 'is \'quoted\''
echo this "should return error
/bin/echo "This is an absolute path test."
/bin
./no_exec.sh
echo "a" |
echo "Hello | cat"
echo "b" || echo "abc"
pwd djdjfk
pwd -t
cd jkkl kjkj
cd /jdkfjdkfj
which sort
which cd
which
exit 0 0 0
exit h
nonexistent_command