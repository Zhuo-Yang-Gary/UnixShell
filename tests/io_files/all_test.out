>>my own "test" case which is 'quoted'
>>mysh: syntax error: unterminated quote
>>This is an absolute path test.
>>mysh: is a directory: /bin
>>mysh: no such file or directory: ./no_exec.sh
>>mysh: syntax error: expected command after pipe
>>Hello | cat
>>mysh: syntax error: expected command after pipe
>>pwd: not expecting any arguments
>>pwd: invalid option: -t
>>cd: too many arguments
>>cd: no such file or directory: /jdkfjdkfj
>>/usr/bin/sort
>>cd: shell built-in command
>>usage: which command ...
>>exit: too many arguments
>>exit: non-integer exit code provided: h
>>mysh: command not found: nonexistent_command
>>
>>
